restaurants/src/main/java/com/mbakgun/restaurants/ui/RestaurantsApp.kt

```diff
@@ -0,0 +1,24 @@
+package com.mbakgun.restaurants.ui
+
+import androidx.compose.runtime.Composable
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.rememberNavController
+import com.mbakgun.restaurants.navigation.Screen
+import com.mbakgun.restaurants.ui.screen.RestaurantsActivityScreen
+
+@Composable
+fun RestaurantsApp(){
+    val navigation = rememberNavController()
+    
+    NavHost(
+        navController = navigation,
+        startDestination = Screen.RestaurantsScreen.route
+    ){
+        composable(
+            route = Screen.RestaurantsScreen.route
+        ) {
+            RestaurantsActivityScreen()
```

Comment:
Hi @boranfrkn , 

Since you had renamed root composable in RestaurantsActivity, 
It looks like we can rename RestaurantsActivityScreen with a convenient one like RestaurantsScreen or alike.

Reply:
Hi @mbakgun i renamed that. Thank you!